package.json

```diff
@@ -18,10 +18,10 @@
         "compile-scripts": "tsc -p scripts",
         "not-needed": "node scripts/not-needed.js",
         "update-codeowners": "node scripts/update-codeowners.js",
-        "test-all": "node node_modules/@definitelytyped/dtslint-runner/dist/index.js --path .",
+        "test-all": "node --require source-map-support/register node_modules/@definitelytyped/dtslint-runner/ --path .",
         "clean": "node scripts/remove-empty.js",
-        "test": "dtslint types",
-        "lint": "dtslint types",
+        "test": "NODE_OPTIONS='--require source-map-support/register' dtslint types",
+        "lint": "NODE_OPTIONS='--require source-map-support/register' dtslint types",
```

Comment:
This breaks runs on windows, if you want to set env vars like this you need to use [`cross-env`](https://www.npmjs.com/package/cross-env)...

Reply:
Sent #61113.

Reply:
@jakebailey Thanks!

@dhensby Do you know if plain `env` will work (no `cross-env` dependency)? I tried and `"test": "env NODE_OPTIONS='--require source-map-support/register' dtslint types"` seemed to work for both Bash and Windows cmd?

Reply:
You must have something installed that is providing that.

```
> typescript@4.8.0 env-test E:\work\TypeScript
> env FOO=1234 node -e 'process.env.FOO'

'env' is not recognized as an internal or external command,
operable program or batch file.
```

Reply:
The best solution would be if `tslint` would accept an `-r` or `--require` flag a like how mocha does, so that no third party dependency is needed to do this, but I recognise that's a change that may be quite difficult to make